docs/guides/developing-components/publishing.md

```diff
@@ -57,21 +57,21 @@ Your `package.json` should have both its `main` entry point and `module` entry p
 }
 ```
 
-## Do export element classes
+### Do export element classes
 
 Every custom element is a class extension of `HTMLElement`; e.g. `class MyElement extends HTMLElement {}`. Be sure that you export this class from your package. When starting from `npm init @open-wc` you will see this export in `src/MyElement.js` and referenced by the `index.js` entry point. This will make it possible for both customization of your element via extension (e.g. `class NewElement extends MyElement {}`) as well as for advanced usage of the custom element in Scoped Registries, both [native](https://github.com/w3c/webcomponents/issues/716) (when available) or [synthetic](/scoped-elements/).
 
-## Do export side effects separately
+### Do export side effects separately
 
 A [side effect](<https://en.wikipedia.org/wiki/Side_effect_(computer_science)>) in javascript is created when the script changes state outside of the scope in which it is run. This includes when code in an es module alters values available on the `window` or `document`. A side effect occurs when registering Custom Elements via `customElements.define()`. JS files with side effects should exist separately from your packages entry point. `npm init @open-wc` takes care of this for you by creating a separate `my-element.js` that manages the registration of your custom element.
 
 If you do choose to publish code with side effects as part of your entry point, it should be placed in a separate function that has to explicitly be called by the consumer (e.g. `run()`, `init()`, etc.). This allows your users to determine when these effects are triggered.
 
-## Do import 3rd party node modules with "bare" import specifiers
+### Do import 3rd party node modules with "bare" import specifiers
 
 When referring to 3rd party dependencies in your code, you cannot guarantee that implementing projects rely on the same file structure as you do. To prevent dependency specifiers from causing issues across different environments, be sure to import these dependencies into your project with "bare" import specifiers so that tooling in your users' environment can manage resolving the location of those dependencies; e.g. `import { LitElement } from 'lit-element';`
 
-## Do include file extensions in import specifiers
+### Do include file extensions in import specifiers
```

Comment:
you don't want it the anchor in the left menu?

Reply:
No.. there were way too many with a lot of text and they were pushing the content of the page to the right